src/services/coingecko.ts

```diff
@@ -8,23 +9,34 @@ const {
 const COINGECKO_API = axios.create({baseURL: "https://api.coingecko.com/api/v3"});
 
 async function getCoinPrice(search: string, fiat = currency) {
-    if (!enableCoinGecko)
+    Logger.changeActionName("coingecko")
+    if (!enableCoinGecko){
+      Logger.warn("enableCoinGecko env is disabled")
       return 0;
+    }
   
     const coins = await COINGECKO_API.get(`/coins/list?include_platform=false`).then(value => value.data);
+
+    if(!Array.isArray(coins))
+        Logger.error(coins, "Error to get list coingecko")
```

Comment:
coins is not `Error` if no `Error` is needed, provide `null` -- otherwise, just use `Logger.warn()`